packages/components/forms/src/base-input/BaseInput.tsx

```diff
@@ -109,7 +102,8 @@ function _BaseInput<E extends React.ElementType = typeof DEFAULT_TAG>(
       style={style}
       placeholder={placeholder}
       className={cx(styles.input, iconClassName, className)}
-      value={valueState}
+      value={value}
+      defaultValue={defaultValue}
```

Comment:
We have to pass only one of those, because if we pass value even empty `value=""` it will consider the value, and set the input as controlled, and will not change it's value unless an onChange is also defined

Reply:
@massao WDYT?

Reply:
IMO, we need to pass both, but `value` should be undefined by default